cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHandler.java

```diff
@@ -478,7 +478,7 @@ public JsonElement serialize(StreamProperties src, Type typeOfSrc, JsonSerializa
       json.addProperty("name", src.getName());
       json.addProperty("ttl", TimeUnit.MILLISECONDS.toSeconds(src.getTTL()));
       json.add("format", context.serialize(src.getFormat(), FormatSpecification.class));
-      json.addProperty("threshold", src.getThreshold());
+      json.addProperty("notification_threshold_mb", src.getNotificationThresholdMB());
```

Comment:
The adapter was already there, it is used to serialize the TTL is seconds. Not sure why there is no deserializer though. When changing the TTL, in what unit do people give it?
Are there other places where we use camel case instead of _ ? Is that what we usually use?